package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "katello",
+  "version": "1.0.0",
+  "scripts": {
+    "test": "jest webpack",
+    "test:watch": "jest webpack --watchAll",
+    "test:current": "jest webpack --watch",
+    "format":
+      "prettier --single-quote --trailing-comma es5 --write 'webpack/**/*.js' '*.{js,json}'",
+    "build": "npm run format && npm run lint",
+    "lint": "eslint --fix webpack/index.js || exit 0"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/katello/katello.git"
+  },
+  "bugs": {
+    "url": "http://projects.theforeman.org/projects/katello/issues"
```

Comment:
@ehelms how's that?

Reply:
This where a new Katello issue should be made--
 http://projects.theforeman.org/projects/katello/issues/new

Reply:
I think the standard convention is to point to the issues page rather than the create page. But it's not that critical and a lot of repos don't even have this field.